[2023-11-26 17:59:28,484] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_covid_etl.create_fact_district_daily manual__2023-11-26T17:57:07.249623+00:00 [queued]>
[2023-11-26 17:59:28,494] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_covid_etl.create_fact_district_daily manual__2023-11-26T17:57:07.249623+00:00 [queued]>
[2023-11-26 17:59:28,494] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-11-26 17:59:28,494] {taskinstance.py:1357} INFO - Starting attempt 2 of 7
[2023-11-26 17:59:28,495] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-11-26 17:59:28,508] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): create_fact_district_daily> on 2023-11-26 17:57:07.249623+00:00
[2023-11-26 17:59:28,513] {standard_task_runner.py:52} INFO - Started process 825 to run task
[2023-11-26 17:59:28,518] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_covid_etl', 'create_fact_district_daily', 'manual__2023-11-26T17:57:07.249623+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/get_data.py', '--cfg-path', '/tmp/tmpuv080teh', '--error-file', '/tmp/tmpnuuuk_iu']
[2023-11-26 17:59:28,519] {standard_task_runner.py:80} INFO - Job 143: Subtask create_fact_district_daily
[2023-11-26 17:59:28,598] {task_command.py:369} INFO - Running <TaskInstance: data_covid_etl.create_fact_district_daily manual__2023-11-26T17:57:07.249623+00:00 [running]> on host 79a606f4be36
[2023-11-26 17:59:28,667] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=oce
AIRFLOW_CTX_DAG_ID=data_covid_etl
AIRFLOW_CTX_TASK_ID=create_fact_district_daily
AIRFLOW_CTX_EXECUTION_DATE=2023-11-26T17:57:07.249623+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-11-26T17:57:07.249623+00:00
[2023-11-26 17:59:28,668] {python.py:173} INFO - Done. Returned value was: None
[2023-11-26 17:59:28,683] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_covid_etl, task_id=create_fact_district_daily, execution_date=20231126T175707, start_date=20231126T175928, end_date=20231126T175928
[2023-11-26 17:59:28,728] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-11-26 17:59:28,780] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
